### PR TITLE
fix(zeebe): allow selection of message start event

### PR DESCRIPTION
**Pull request by marstamm**
related to https://camunda.slack.com/archives/GP70M0J6M/p1648052322348169

### CODE REVIEW

File: test/spec/provider/zeebe/MessageProps.spec.js [hunk 1]
```diff
@@ -89,6 +89,23 @@ describe('provider/zeebe - MessageProps', function() {
     }));
 
 
+    it.only('should not display for Message <None>', inject(async function(elementRegistry, selection) {
```
Comment (barmac) [inline test/spec/provider/zeebe/MessageProps.spec.js:92]:
```suggestion
    it('should not display for Message <None>', inject(async function(elementRegistry, selection) {
```

Comment (marstamm) [reply test/spec/provider/zeebe/MessageProps.spec.js]: > How come the checks have passed 😆

I have no Idea 😄

Fixed with [6ed77f4](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/623/commits/6ed77f464894db8b2c2970ac0cc3c143274fcbdc)

**Comment by barmac** [comment]
This works for message start event but still breaks for intermediate message event. I think we should fix them all in this PR.

**Comment by barmac** [comment]
```

ExtensionElementsUtil.js?fc52:15 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'get')
    at getExtensionElementsList (ExtensionElementsUtil.js?fc52:15:44)
    at getSubscriptions (MessageProps.js?6966:219:53)
    at getSubscription (MessageProps.js?6966:213:25)
    at getCorrelationKey (MessageProps.js?6966:206:24)
    at getValue (MessageProps.js?6966:105:12)
    at TextfieldEntry (index.esm.js?9a4a:1553:1)
    at p.SubscriptionCorrelationKey [as constructor] (MessageProps.js?6966:171:24)
    at p.M [as render] (preact.module.js?cb16:1:8038)
    at I (preact.module.js?cb16:1:5744)
    at m (preact.module.js?cb16:1:2155)
```

**Comment by marstamm** [comment]
When you break something in a fix that worked before 😅 

The conditions were broken, of course we need a message for a SubscriptionCorrelationKey [924cb5a](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/623/commits/924cb5a2da33eb34ffe4aee2bb1b45eed36c2066)

**Comment by nikku** [comment]
And message end events work, too?

File: src/provider/zeebe/properties/MessageProps.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ export function MessageProps(props) {
     );
   }
 
-  if (canHaveSubscriptionCorrelationKey(element)) {
+  if (message && canHaveSubscriptionCorrelationKey(element)) {
```
Comment (nikku) [inline src/provider/zeebe/properties/MessageProps.js:47]: Add test case that verifies this fails without.

**Comment by barmac** [comment]
> And message end events work, too?

They do, but that's probably because one cannot configure message for a message end event 🤡 
image

**Review by barmac — APPROVED** [review]

**Comment by nikku** [comment]
I did integration test this one on `main` and it just works. Will go a head and release this!

**Pull request merged**